### ISSUE TITLE
iTunes Match Library songs in playlist not showing

### ISSUE THREAD

**Issue by GRMKJ** [ciderapp/Cider-2#221]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://cider.sh/legal/conduct) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ciderapp/Sabiiro-BugTracker/issues) for a bug report that matches the one I want to file, without success.
- [X] I have read the [troublshooting FAQs](https://cider.gitbook.io/welcome-to-gitbook/support/faqs) and none solved my issue.

### What operating system are you using?

Windows

### Build Details

5/16/2023

### Operating System Version

Windows 11 version 22H2

### Describe the Bug

Cider 2 seems to not load songs from iTunes Match 

![image](https://github.com/ciderapp/Sabiiro-BugTracker/assets/22628642/c8dac0db-d6df-4a75-a6a1-b931233636ad)

on the otherside Cider 1.6 dont hace an issue with this

![image](https://github.com/ciderapp/Sabiiro-BugTracker/assets/22628642/87ab21b1-cace-400e-bda4-52874ad70b87)

### Steps to Reproduce

1. Make a Playlist in any client
2. Add iTunes Match songs
3. Try to play the playlist on Cider 2

**Comment by godssonisme** [comment]
I can agree on this.
Within iTunes AND Cider it shows your "Library" and "Apple Music". Some items from my Libray that are exclusive to **iTunes Match** don't show within Cider.
Interestingly enough, my unique music (my created music stored in iTunes Match) is showing and playing Cider perfectly.

But my example of the issue is here:
![image](https://github.com/ciderapp/Cider-2/assets/65793801/ae87f175-180a-4c4a-9727-18d95b9ddf3d)
![image](https://github.com/ciderapp/Cider-2/assets/65793801/4c9aeff5-e628-4910-87f1-f473a432acb0)

I played this Usher album from my Library (it is a unique extended version that was only on vinyl, that I ripped to iTunes Match) which is 5:37. But Cider plays the new, abridged version of the song that is 4:26, and not even in my Library.
It's some new edition:
![image](https://github.com/ciderapp/Cider-2/assets/65793801/42c031e5-1712-489d-a5c7-c5deccffa3d1)

More over, I noticed it's playing all clean version of songs as well.
